pkg/types/openstack/machinepool.go

```diff
@@ -1,5 +1,15 @@
 package openstack
 
+// AllowedServerGroupPolicies is the set of allowed values for the
+// "ServerGroupPolicy" property of the MachinePool. The value will be set on
+// the Nova server group that will contain the machines.
+var AllowedServerGroupPolicies = map[ServerGroupPolicy]struct{}{
+	"affinity":           {},
```

Comment:
Can we make these constants of type `ServerGroupPolicy`? That way the constant can be used in pkg/tfvars/openstack/openstack.go, too.